ONSdigital/census-rm-ddl
# Rmc 156 c woc field

# Motivation and Context
Add Field WOC action rules to existing 2019 census plan

# What has changed
Added in new field rules as referenced in https://collaborate2.ons.gov.uk/confluence/display/SDC/RM+Census+Run+Book
Re ordered the manual scripts directory, into schema and actionplan/rules.  Renamed files appropriately

# How to test?
1. Can you execute the new SQL in action rules in your local postgres, you will need to run 0001-create-ICL-IPQ-action-rules.sql 1st to set up the action rule, you can just copy/run the action rule insert.  
Then run the sql in 0003-woc-fieldwork-tranches.sql.  This should cause no errors and set up the new rules in your table.

2.  Do the new treatment codes and dates look correct based on the linked run book 

# Links
https://trello.com/c/Xc0Ufor0/947-rmc-156c-script-to-set-up-action-rules-to-generate-woc-field-visits-3
